.github/workflows/tests.yaml

```diff
@@ -17,37 +17,52 @@ jobs:
       matrix:
         include:
           # Django 2.2
-          - django: "2.2"
-            python: "3.6"
-          - django: "2.2"
-            python: "3.7"
           - django: "2.2"
```

Comment:
Since we've updated the Django version in `requirements.txt` to `Django==4.2` we can remove the envs for `Django<4.2` as well.

Reply:
We aren't using requirements.txt here